### PR TITLE
Ignore trailing whitespace when addressing the bot by nick at end

### PR DESCRIPTION
**Pull request by matiasw**
Seems to fix https://github.com/progval/Limnoria/issues/1455 

Again, if you decide to merge, please add the `hacktoberfest-accepted` label.

### CODE REVIEW

**Comment by progval** [comment]
Could you add a test for this?

**Comment by matiasw** [comment]
Sure; I added it under the Misc plugin, as that's where this was originally reported.

**Comment by progval** [comment]
It should be in `tests/test_callbacks.py`, that's where the other tests for this are

**Comment by matiasw** [comment]
> It should be in `tests/test_callbacks.py`, that's where the other tests for this are

How do I run these tests? `supybot-test` seems to be only for plugins, and there is no `test` target in the Makefile.

**Comment by progval** [comment]
`supybot-test test`

> there is no test target in the Makefile.

There is, and it runs both the above and plugin tests

**Comment by matiasw** [comment]
Ok; I moved & updated the test.

File: test/test_callbacks.py [hunk 1]
```diff
@@ -269,9 +269,16 @@ def testAddressedWithNickAtEnd(self):
         irc = getTestIrc()
         msg = ircmsgs.privmsg('#foo', 'baz, bar')
         irc._tagMsg(msg)
+        nick = irc.nick
```
Comment (progval) [inline test/test_callbacks.py:272]:
```suggestion
```

Comment (matiasw) [reply test/test_callbacks.py]: Sorry! ;(

**Comment by progval** [comment]
thx

**Pull request merged**